src/main/java/com/example/checkstylereviewdogsample/utils/SampleUtils.java

```diff
@@ -0,0 +1,7 @@
+package com.example.checkstylereviewdogsample.utils;
+
+public class SampleUtils {
+    public static String getHelloWorld() {
+        return "Hello World";
+    }
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def rcurly' has incorrect indentation level 4, expected level should be 2.